apps/sage/account/NanoList.tsx

```diff
@@ -38,41 +24,71 @@ export default function NanoList() {
         a.href = window.URL.createObjectURL(data)
         a.download = 'registration.zip'
         a.click()
-      }).catch(err => console.log(err.message))
+      }).catch(err => {
+        if (err.message) {
```

Comment:
This makes me wonder if it's possible for there to _not_ be a `err.message`?  If that is the case, the user wouldn't know what went wrong.  :(